File: test6.py
def fizzbuzz(n):
    lst = []
    count = 0

    for i in range(n):
        count += 1
        lst.append(count)
        
        if lst[i] % 3 == 0 and lst[i] % 5 == 0:
            lst[i] = 'FizzBuzz'
        elif lst[i] % 3 == 0:
            lst[i] = 'Fizz'
        elif lst[i] % 5 == 0:
            lst[i] = 'Buzz'

    return(lst)

fizzbuzz(15)

File: test6.js
// Return an array
function fizzbuzz(n){
  //
  let arr = []
  let count = 1

  for(let i = 0; i < n; i++){
    arr.push(count++)
    if(arr[i]%5 === 0 && arr[i]%3 === 0){
        arr[i] = 'FizzBuzz'
    }
    else if(arr[i]%3 === 0) {
        arr[i] = 'Fizz'
    } else if(arr[i]%5 === 0){
        arr[i] = 'Buzz'
    }
  }
  return arr
}

fizzbuzz(15)